setup.py

```diff
@@ -196,4 +196,16 @@ def run(self):
         "License :: OSI Approved :: MIT License",
         "Programming Language :: Python :: 3 :: Only",
     ],
+    extras_require={
+        'passagemath': [
+            'passagemath-combinat',
+            'passagemath-flint',
+            'passagemath-modules',
+            'passagemath-plot',
+            'passagemath-repl',
+            'passagemath-symbolics',
+            'networkx',
+            'sympy',
+        ],
+    },
 )
```

Comment:
Could you add a job to github actions to test that installing with the passagemath extra works? I can do this instead when I get time if you're not too familiar with github actions.

Reply:
I've added it to test.yml